package-lock.json

```diff
@@ -2969,18 +2969,26 @@
             }
```

Comment:
Hey @krobd, just wondered why the package-lock version field hasn't bumped along with the package.json, could we give `npm i` another whirl?

Reply:
Hi @cgwyllie - just missed this from the commit list, updated now 👍